setup/gunstar-maker.py

```diff
@@ -59,7 +59,7 @@
 engines["suricata301"] = {"type":"suricata", "version":"3.0", "eversion":"3.0.1"}
 engines["suricata31"] = {"type":"suricata", "version":"3.1", "eversion":"3.1"}
 engines["suricata311"] = {"type":"suricata", "version":"3.1", "eversion":"3.1.1"}
-engines["suricata32"] = {"type":"suricata", "version":"3.2", "eversion":"3.2.0"}
```

Comment:
@travisbgreen - just wondering if you confirmed this is working as expected on adding/running tests with  3.2(0) Suricata engine?

Reply:
afaik was just to make naming consistent for 3.2

Reply:
If it has no other (unforeseen) effect - good with me :)